### PR TITLE
refactor(): icon

### PR DESCRIPTION
**Pull request by dan-entous**
[UWEB-6539](https://jira.ultimaker.com/browse/UWEB-6539)

## purpose
* simplify the component
    * remove orphaned  properties & logic
    * add the svg icon set to the html page instead of loading it via a url
    * remove the dependency on a url in the icon’s href reference
    * update the component format
* add storybook icon stories for each icon

## playground
* https://playground-refactor-icon.ultimaker.k8s-dev.ultimaker.works

### CODE REVIEW

File: src/js/components/atoms/icon/icon.ts [hunk 1]
```diff
@@ -1,37 +1,34 @@
...
-        return `${this.iconUrl}#icon-${this.iconName}`;
+    public get iconHref(): string {
+        return `${this.iconSetUrl}#icon-${this.iconName}`;
     }
 
-    mounted() {
-        // Only render in browser
-        this.ready = true;
+    /**
+     * prevents icons from being rendered on server side
```
Comment (jasperkuipers) [inline src/js/components/atoms/icon/icon.ts]: maybe rewrite this comment...

File: src/js/components/atoms/icon/stories/icon.html [hunk 2]
```diff
@@ -0,0 +1,2 @@
+<!-- @format -->
+<icon v-bind="$props" style="font-size: 32px;" />
```
Comment (jasperkuipers) [inline src/js/components/atoms/icon/stories/icon.html]: i don't like inline styles, can't we get rid of it?

Comment (dan-entous) [reply src/js/components/atoms/icon/stories/icon.html]: these are just for storybook

File: src/js/components/atoms/icon/icon.ts (hunk 1 above)
Comment (dan-entous) [reply src/js/components/atoms/icon/icon.ts]: this code has been removed

File: src/js/components/atoms/icon/stories/icon.html (hunk 2 above)
Comment (jasperkuipers) [reply src/js/components/atoms/icon/stories/icon.html]: 👍

File: src/js/components/atoms/icon/icon.ts (hunk 1 above)
Comment (jasperkuipers) [reply src/js/components/atoms/icon/icon.ts]: 👍

**Pull request merged**